src/components/grapher/GrapherDots.vue

```diff
@@ -1,40 +1,37 @@
 <template>
   <g class="grapher-dots--dots-container">
-    <circle
-      v-for="dot in stuntSheetDots"
+    <Dot
+      v-for="(dot, index) in stuntSheetDots"
       :key="`${dot.x}-${dot.y}-dots--dot`"
       class="grapher-dots--dot"
+      data-test="grapher-dots--dot"
+      :transform="`translate(${dot.xAtBeat(beat - 1)}, ${dot.yAtBeat(
+        beat - 1
+      )})`"
       :cx="dot.xAtBeat(beat - 1)"
```

Comment:
Is cx and cy doing anything as props?

Reply:
Nope, it's just so that the tests can see where the dot's being rendered.

Reply:
If they aren't doing anything, it's okay to omit the tests! Let's not add code specifically for tests when possible.